src/many-ledger/src/module/ledger_mintburn.rs

```diff
@@ -9,7 +9,14 @@ use many_modules::events::EventInfo;
 use many_modules::ledger;
 use many_modules::ledger::{TokenBurnArgs, TokenBurnReturns, TokenMintArgs, TokenMintReturns};
 use many_types::ledger::Symbol;
+use once_cell::sync::Lazy;
 use std::collections::BTreeSet;
+use std::str::FromStr;
+
+// Production network MFX address
+pub static MFX: Lazy<Address> = Lazy::new(|| {
+    Address::from_str("mqbh742x4s356ddaryrxaowt4wxtlocekzpufodvowrirfrqaaaaa3l").unwrap()
```

Comment:
The production network MFX address. Won't work if the MFX address is something else.